applications/serial_lte_modem/doc/slm_data_mode.rst

```diff
@@ -64,30 +64,30 @@ If the current sending function succeeds and :ref:`CONFIG_SLM_DATAMODE_URC <CONF
 The ``<size>`` value is a positive integer.
 This Unsolicited Result Code (URC) can also be used to impose flow control on uplink sending.
 
+.. note::
+  If the data mode operation fails, the SLM application moves to a state where the data received from UART is dropped until the MCU sends the termination command.
```

Comment:
I guess that it could be a fails due to a network problem. However, this applies to all the datamode operations. The bullet point list applies to particular scenarios with proxy clients.

Reply:
So here it's only data sending? If so, this could be made more specific.
Also, shouldn't this sentence also mention the sending of the termination string by SLM, just as the other explanation below?

Reply:
There is only data sending. The termination strin reference could be added, yes.

Reply:
And the beginning changed to something like `If the sending operation fails while in data mode`?